docs/docsite/rst/moddev_guide_deps.rst

```diff
@@ -0,0 +1,75 @@
+..
+  Copyright (c) Ansible Project
+  GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+  SPDX-License-Identifier: GPL-3.0-or-later
+
+.. _ansible_collections.community.general.docsite.guide_deps:
+
+**********************************
+``deps`` Guide
+**********************************
```

Comment:
Toplevel keys should use `===`, and the number of equal signs should equal the number of letters.
```suggestion
``deps`` Guide
==============
```

Reply:
I copied that from `guide_packet.rst`, also a top level

Reply:
That got vendored from ansible-core, I guess it needs fixing. (Some of the other guide_ files are also like that...)